setup.py

```diff
@@ -41,6 +41,7 @@
     'tmt/steps/execute',
     'tmt/steps/report',
     'tmt/steps/finish',
+    'tmt/test_checks',
```

Comment:
A bit of :bike: shedding :)

@happz should we just not call this `checks`? I prefer one word wording, and I like `tmt checks` :)
And also this nicely alligns with the name in the `tmt` metadata, so less confusing :)

Reply:
Probably. I'm just not very happy about the generic name, "checks" can mean pretty much anything. Maybe we can find a place where it can be "checks" while not allowing to be mistaken for "prepare checks". Maybe we could add `tmt.test` subpackage, to hold `checks` and `frameworks` (from `tmt.frameworks`, but it's not for `prepare` or `report`, it's "test frameworks"...)

Reply:
Let's see what others think, but yes, `tmt.test.checks` would make sense to me

Reply:
@lukaszachy @psss any preferences, ideas, or concerns?

Reply:
we agreed on `checks` on the meeting